lib/nucore.rb

```diff
@@ -108,6 +108,10 @@ def self.included(base)
       end
       module ClassMethods
 
+        def order_by_asc_nulls_first(field)
+          NUCore::Database.oracle? ? order("#{field} asc nulls first") : order(field)
+        end
+
         def order_by_desc_nulls_first(field)
           NUCore::Database.oracle? ? order("#{field} desc nulls first") : order("-#{field}")
```

Comment:
In both these cases we want nulls first. There are sometimes we want descending, i.e. reverse chronological, but put nulls first (we're doing this in the transaction search). Then in this case, we need nulls first because we want the parent-level order details at the top, but other than that we want normal ascending order.

The default for oracle and mysql are opposite. In oracle, `desc` has the nulls first by default. We could conceivably take it out here, but I wanted to make it clear what we're choosing to do, and not relying on the default (at least in oracle's case).